### PR TITLE
[OSDEV-851] Checkout tfvars from ci-deployment

### PR DESCRIPTION
**Pull request by killev**
Place 'terraform.tfvar' files to repository and move sensitive info to private repository opensupplyhub/ci-deployment/

[OSDEV-851](https://opensupplyhub.atlassian.net/browse/OSDEV-851)

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

File: .github/workflows/deploy_to_aws.yml [hunk 1]
```diff
@@ -44,10 +44,16 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Copy tfvars from S3 bucket for ${{ vars.ENV_NAME }}
+      - name: Checkout config repository
+        uses: actions/checkout@v4
+        with:
+          repository: 'opensupplyhub/ci-deployment'
+          path: 'terraform-config'
```
Comment (eugen4ernov) [inline .github/workflows/deploy_to_aws.yml:51]: can't see 'terraform-config' in 'ci-deployment'

Comment (killev) [reply .github/workflows/deploy_to_aws.yml:51]: This is path in existing folder not path In the repository. I.e. it's the second argument in the `git clone` command:  `git clone  git://...res.git terraform-config`

**Review by roman-stolar — APPROVED** [review]

File: deployment/environments/terraform-development.tfvars [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+bastion_ami = "ami-0bb3fad3c0286ebd5"
+bastion_instance_type = "t3.nano"
+
+rds_allocated_storage = "128"
+rds_engine_version = "12"
+rds_parameter_group_family = "postgres12"
+rds_instance_type = "db.t3.micro"
+rds_database_identifier = "opensupplyhub-enc-stg"
+rds_database_name = "opensupplyhub"
```
Comment (eugen4ernov) [inline deployment/environments/terraform-development.tfvars:22]: could that be looked up using **data** or **module.database_enc** or templated by variables like **${var.project}-${var.environments}-db** in TF code?

Comment (killev) [reply deployment/environments/terraform-development.tfvars:22]: Probably yes, but this is definitely different task

Comment (killev) [reply deployment/environments/terraform-development.tfvars:22]: Also what for? what's advantage of using different database names for different environments?

Comment (eugen4ernov) [reply deployment/environments/terraform-development.tfvars:22]: Just for the recognition while you parse some logs. If you don't want this - you can skip "var.env" part

Comment (killev) [reply deployment/environments/terraform-development.tfvars:22]: https://opensupplyhub.atlassian.net/browse/OSDEV-937

**Review by roman-stolar — APPROVED** [review]

**Review by vladsha-dev — APPROVED** [review]

**Pull request merged**